build

```diff
@@ -21,14 +21,13 @@ function shell(cmd) {
 }
 
 // Check to make sure the most recently added or updated dependency is installed at the correct version
-try {
-	var version = require('esbuild').version.split('.');
-	if (parseInt(version[1]) < 16) {
-		throw new Error("esbuild version too old");
+for (var pkg of ['oxc-transform', 'fast-glob', 'typescript', 'sucrase']) {
+	try {
+		require.resolve(pkg);
+	} catch (e) {
+		console.log('Installing dependencies...');
+		shell('npm install ' + pkg);
```

Comment:
We usually just put the most recent added/updated package here, instead of needing to loop through multiple of them.